wickedone/phrase-tag-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "WickedOne\\PhraseTagBundle\\WickedOnePhraseTagBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    }
```

Comment:
Just a suggestion, you can offer to configure DSN as well.
![image](https://github.com/user-attachments/assets/afbb5e67-9ede-4d3a-8b1c-16f4ad2d226d)

```suggestion
    "copy-from-recipe": {
        "config/": "%CONFIG_DIR%/"
    },
    "env": {
        "PHRASE_DSN": "phrase://PROJECT_ID:API_TOKEN@default?userAgent=myProject"
    }
```